pkgs/tools/text/reckon/gemset.nix

```diff
@@ -0,0 +1,12 @@
+{
+  chronic = {
+  };
+  fastercsv = {
+  };
+  highline = {
+  };
+  reckon = {
+  };
+  terminal-table = {
+  };
```

Comment:
Turns out when you run bundix in a nix-shell it needs the ssl cert export patch applied to the shell or bundix cant download things. This is fixed in latest commit.